src/Lib/Types/Manifest.hs

```diff
@@ -79,6 +109,13 @@ testManifest =
     "short": "Create Tor websites, hosted on your Embassy.",
     "long": "Embassy Pages is a simple web server that uses directories inside File Browser to serve Tor websites."
   },
+  "hardware-requirements" {
+    "device": {
+      "processor": "intel",
+      "display": "r'^{.*}$'"
+    },
+    "ram": "8"
```

Comment:
this package requires at least 8 whole bytes of ram! Wow!

Reply:
😅 also i have fixed that both keys are regex